linkcheck/configuration/__init__.py

```diff
@@ -302,14 +302,6 @@ def sanitize_loginurl(self):
         """Make login configuration consistent."""
         url = self["loginurl"]
         disable = False
-        if not self["loginpasswordfield"]:
-            log.warn(LOG_CHECK,
-                     _("no password input name attribute given for login URL."))
-            disable = True
-        if not self["loginuserfield"]:
-            log.warn(LOG_CHECK,
-                     _("no username input name attribute given for login URL."))
-            disable = True
```

Comment:
Hm.  Indecision strikes again: maybe it's good to warn the user if the field names are not configured -- in case they're trying to configure login form handling and forgot a step.  Just not disable in that one use-case where we want to support login-less but password-ful login forms (aka mailman moderation pages).

I'm also thinking that maybe if loginuserfield and loginpasswordfiel are _both_ missing, maybe we want to warn _and_ disable?  Does a login form that doesn't accept any credentials make sense?  (Actually, maybe it does!  Maybe you just need to acknowledge something, I dunno, EU cookie warning interstitial page or something?)

Reply:
> The checks for empty loginpasswordfield and loginuserfield are always going to pass. They have default values and you are prevented from resetting them. Let's get rid of them to avoid confusion.

I forgot this bit.  Yeah, makes sense.